services/galley/src/Galley/API.hs

```diff
@@ -392,6 +407,8 @@ sitemap = do
       description "Max. number of conversations to return"
   ---
 
+  -- This endpoint can lead to the following events being sent to clients:
```

Comment:
when you say clients, dont you mean users? im a bit confused by this terminology